helm/microgateway/requirements.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+dependencies:
```

Comment:
New line missing at the end

Reply:
Fixed https://github.com/wso2/kubernetes-microgateway/pull/1/commits/1b4557f31cfbe9f0c00d0b561bde967a6ab758a7